torch_glow/src/PyTorchCommon.cpp

```diff
@@ -213,7 +217,23 @@ glow::Type ptTypeToGlowType(const c10::TensorType &ptType) {
   }
 
   auto scalarType = ptType.scalarType().value();
-  return glow::Type(scalarTypeToElemKind(scalarType), dims);
+  auto glowElemKind = scalarTypeToElemKind(scalarType);
+  return glow::Type(glowElemKind, dims);
+}
+
+glow::Type ptQTypeToGlowQType(const c10::TensorType &ptType, const float &scale,
```

Comment:
Imo, instead of duplicating this, maybe let's just delete ptTypeToGlowType and ptQTypeToGlowQType and do this in ptTensorToGlowTensor since 1. these are only used there and 2. TensorType doesn't contain all the needed info to build a glow type since scale and zero point are missing